test/jdk/java/foreign/CallGeneratorHelper.java

```diff
@@ -213,11 +213,7 @@ public static void main(String[] args) {
     static void generateDowncalls(boolean header) {
         if (header) {
             System.out.println(
-                "#ifdef _WIN64\n" +
-                "#define EXPORT __declspec(dllexport)\n" +
-                "#else\n" +
-                "#define EXPORT\n" +
-                "#endif\n"
+                "#include \"export.h\"\n"
```

Comment:
We don't generate these header files any more, so the changes to this file are not really needed.

Reply:
I still wouldn't like to keep the bad hard-coded defines. Are you okay with me pushing these changes, and then you can remove the parts of the test that are not actually used anymore? If the code is not used it should not matter much to you either way.

(I mean I could back out these changes, but then we'd have the bad code in place while waiting for you to remove it, putting pressure on you to actually remove it.)

Reply:
Yes, that's fine. Sorry, I meant to file a JBS issue and come back with another comment last time, but I got distracted. Filed: https://bugs.openjdk.org/browse/JDK-8327994 now